docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# How to Contribute
+
+Please refer to the [readme](https://github.com/tablexi/nucore-open/blob/master/README.md) for spin-up information and basic context. 
+
+## Opening a Pull Request
+
+Once you're ready to open a pull request, please follow our pull request template for your PR description. Please include tests for any new functionality added. Refer to our [Coding Standards](https://github.com/tablexi/nucore-open/blob/master/doc/coding_standards.md) for information on style guidelines and best practices. Once your PR is approved, please squash commits and paste the "Release Notes" section from the PR description as the commit message for the PR.
```

Comment:
Once your PR is approved, please use the "Squash and Merge" option and paste the "Release Notes"...